src/hooks/keywordSearch.test.ts

```diff
@@ -0,0 +1,59 @@
+import Cookies from 'js-cookie';
+import nock from 'nock';
+import { StatusCodes } from 'http-status-codes';
+import { List } from 'immutable';
+import { buildGetItemsByKeywordRoute } from '../api/routes';
+import { mockHook, setUpTest } from '../../test/utils';
+import { ITEMS, UNAUTHORIZED_RESPONSE } from '../../test/constants';
+import { buildSearchByKeywordKey } from '../config/keys';
+import { Item } from '../types';
+
+const { hooks, wrapper, queryClient } = setUpTest();
+jest.spyOn(Cookies, 'get').mockReturnValue({ session: 'somesession' });
+
+describe('Keyword Search Hook', () => {
+  afterEach(() => {
+    nock.cleanAll();
+    queryClient.clear();
+  });
+
+  describe('useKeywordSearch', () => {
+    const range = 'all';
+    const keywords = 'cat&dog'
```

Comment:
That's because it's explorer's search which join with a `&` right? I'm wondering if it wouldn't be more practical for the front to return an array, and then the back/queryclient join them. Let me know what you think.

Reply:
I actually thought about both ways, and in my opinion, a single formatted string is better for following reasons: If the front end returns an array, it needs to return extra information about if the user is searching for the union or intersection of the array (an extra parameter). Besides, the Postgresql could automatically process spaces (' cat & dog ' also works in this case), so we could directly take the input from the search box (from front-end), and just pass down, without extra processing. But if we want the front-end to send back an array of strings, we need to slice them, and then join them again.
Let me know if you have other concerns about this.